EventPlanning.BL/EventManager.cs

```diff
@@ -37,6 +37,27 @@ public List<Event> GetUserEvents(int userId)
             return repository.GetUserEvents(userId)?.ToList();
         }
 
+        /// <summary>
+        /// Returns events where any event property contains pattern
+        /// </summary>
+        /// <param name="pattern"></param>
+        /// <returns></returns>
+        public List<Event> GetEvents(string pattern)
+        {
+            return repository.GetEvents(pattern)?.ToList();
+        }
+
+        /// <summary>
+        /// Returns events between date intervals
+        /// </summary>
+        /// <param name="dateFrom"></param>
+        /// <param name="dateTo"></param>
+        /// <returns></returns>
+        public List<Event> GetEvents(DateTime dateFrom, DateTime? dateTo)
+        {
+            return repository.GetEvents(dateFrom, dateTo ?? dateFrom).ToList();
```

Comment:
What happens if `dateFrom` is sent as `2017-01-29 00:00:00`  which is a default `date` that is sent most of the time from Front end (no hours selected is defaulted as `00:00:00`